services/ui-src/src/utils/testing/mockFullReportJSON.tsx

```diff
@@ -28,7 +28,7 @@ export const mockFullReportJSON: ReportJson = {
               props: {
                 href: "https://www.govinfo.gov/content/pkg/PLAW-109publ171/pdf/PLAW-109publ171.pdf",
                 target: "_blank",
-                ariaLabel: "Link opens in new tab",
+                "aria-label": "Link opens in new tab",
```

Comment:
suggestion: include visual content in label so it gets a full readout, or move to separate PR